dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -276,6 +281,22 @@ def change_config(python, backend, cheatsheet, asciiart):
         console.print(f'[blue]Backend cached_value {backend}')
 
 
+@option_verbose
+@main.command(name="static-check")
+@click.option('-t', '--type', type=click.Choice(PRE_COMMIT_LIST), multiple=True)
+def static_check(verbose: bool, type: Tuple[str]):
+    if check_package_installed('pre_commit'):
+        command_to_execute = ['pre-commit', 'run']
```

Comment:
I think it should be worthwile to add `--color always` (otherwise no colors are generated when you run it with stdout redirection)

Reply:
Or maybe simply - this will work out-of-the box if you do not use "capture-output" in the next step. This way it will write dirrectly to stdout and not only it will be printed "as it comes" but also it will automatically get colored ouput.

Reply:
@potiuk  I tried running without "capture-output". But it didn't get colored output. Am I missing something?
<img width="1436" alt="Screenshot 2022-01-24 at 7 51 32 PM" src="https://user-images.githubusercontent.com/10162465/150800246-ceff7c35-43dd-44f5-89de-405b046a6740.png">